addons/website_sale/static/src/js/website_sale.editor.js

```diff
@@ -238,4 +239,24 @@ options.registry.productsSearchBar = options.Class.extend({
         }).open();
     },
 });
+
+sAnimations.registry.websiteSaleCurrency = sAnimations.Class.extend({
+    selector: '.oe_website_sale',
+    read_events: {
```

Comment:
@kea14 You are using read_events and then says in the method "if (editableMode)"... that will never happen. Use 'edit_events' and no need of `if`. You can also test the feature you adapt to see if it is still working...

Reply:
Thanks. I always test to see if it is still working but here I don't understand what it is supposed to do

Reply:
Nvm, it is working